### PR TITLE
Feature/backfill member accounts

### PR DESCRIPTION
**Pull request by vonElfvin**

### CODE REVIEW

File: src/main/java/com/hedvig/productPricing/service/serviceIntegration/accountService/AccountService.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.hedvig.productPricing.service.serviceIntegration.accountService;
+
+import org.springframework.http.ResponseEntity;
+
+public interface AccountService {
+  ResponseEntity<?> createAccount(String memberId);
```
Comment (vonElfvin) [inline src/main/java/com/hedvig/productPricing/service/serviceIntegration/accountService/AccountService.java]: Any suggestions?

Account-service returns "ResponseEntity<Void>

File: src/main/java/com/hedvig/productPricing/service/serviceIntegration/accountService/AccountServiceClient.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
+package com.hedvig.productPricing.service.serviceIntegration.accountService;
+
+import com.hedvig.productPricing.config.FeignConfiguration;
```
Comment (vonElfvin) [inline src/main/java/com/hedvig/productPricing/service/serviceIntegration/accountService/AccountServiceClient.java]: Note the location of the FeignConfiguration, found similar usage in other services. TODO: Move to a central config folder?

File: src/main/java/com/hedvig/productPricing/service/serviceIntegration/accountService/AccountService.java (hunk 1 above)
Comment (palmfjord) [reply src/main/java/com/hedvig/productPricing/service/serviceIntegration/accountService/AccountService.java]: Does `ResponseEntity<Void>` not work here then?

File: src/main/java/com/hedvig/productPricing/service/accounts/AccountEventListener.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  public AccountEventListener(
+    AccountService accountService
+  ) {
+    this.accountService = accountService;
+  }
+
+  @EventHandler
+  public void on(ContractSignedEvent e) {
+    String memberId = e.getMemberId();
+    if (!BLACKLISTED_MEMBERS.contains(e.getMemberId())) {
```
Comment (palmenhq) [inline src/main/java/com/hedvig/productPricing/service/accounts/AccountEventListener.java]: i think we should unnegate this and make an early return to keep the indentation level a bit smaller :)

**Pull request merged**